docs/deployment/uwsgi_cookiecutter_nginx.rst

```diff
@@ -0,0 +1,233 @@
+      #
+      # --chmod-socket=020                  # Change permissions on socket to
+      #                                       at least 020 so that in combination
+      #                                       with "--gid www-data", Nginx will be able
+      #                                       to write to it after  uWSGI creates it
+      #
+      # --enable-threads                    # Execute threads that are in your app
+      #
+      # --plugin=python3                    # Use the python3 plugin
+      #
+      # --socket ~/myproject/tmp/myproject.sock   # Where to put the unix socket
+      #
+      # --manage-script-name
+      #
+      # --mount /=wsgi:app                  # Mount the path "/" on the symbol
+      #                                       "app" found in the file wsgi.py
+      #
+      # --uid ubuntu                        # masquerade as the ubuntu user
+      #
+      # --gid www-data                      # masquerade as the www-data group
+      #
+      # --virtualenv env                    # Use packages installed in your venv
+
+#.  Verify that the output of the previous step includes a line that looks approximately like this:
+
+    .. code-block:: bash
+
+       WSGI app 0 (mountpoint='/') ready in 1 seconds on interpreter 0x5615894a69a0 pid: 8827 (default app)
+
+    If any errors occurred, you will need to correct them. If you get a
+    ``callable not found or import error``, make sure you your ``--mount
+    /=wsgi:app`` matches the ``app`` symbol in the ``wsgi.py`` file. An import
+    error that looks like ``ImportError: No module named 'wsgi'`` probably
+    indicates a mismatch in your --mount arguments. Any other import errors
+    probably means that the package it's failing to import either is not
+    installed or is not accessible by the user. That's why we chose to
+    masquerade as the normal user that you log in as, so you would for sure
+    have access to installed packages.
+
+#.  Add a new file at ``/etc/nginx/sites-enabled/myproject.conf`` with
+    the following contents. Also change any occurrences of the word ``ubuntu``
+    to your actual username.
+
+    .. code-block:: nginx
+
+      server{
+        server_name _;
+
+        root /home/ubuntu/myproject/;
+
+        location /  {
+          include uwsgi_params;
+          # The socket location must match that used by uWSGI
+          uwsgi_pass unix:/home/ubuntu/myproject/tmp/myproject.sock;
+        }
+
+      }
+
+
+#   If there is a file that is at /var/nginx/sites-enabled/default,
+    remove it so your new nginx config file will catch all traffic.
+    (If ``default`` is in use and important, simply add a real
+    ``server_name`` to ``/etc/nginx/sites-enabled/myproject.conf``
+    to disambiguate them.
+
+#.  Reload Nginx
+
+    .. code-block:: bash
+
+       $ sudo nginx -s reload
+
+#.  Visit ``http://localhost`` in a browser. Alternatively, call ``curl localhost``
```

Comment:
Suggest removing backticks so that http://localhost renders as a clickable link.